manifest.yml

```diff
@@ -6,16 +6,20 @@ applications:
     buildpacks:
       - nodejs_buildpack
     env:
-      AUTH_BASE: ((AUTH_BASE))
+      AUTH_BASE: https://uat.hsesinfo.org
       AUTH_CLIENT_ID: ((AUTH_CLIENT_ID))
       AUTH_CLIENT_SECRET: ((AUTH_CLIENT_SECRET))
       NEW_RELIC_APP_NAME: tta-smarthub-((env))
       NEW_RELIC_LICENSE_KEY: ((NEW_RELIC_LICENSE_KEY))
-      NODE_ENV: ((NODE_ENV))
+      NODE_ENV: production
+      POSTGRES_DB: ((POSTGRES_DB))
+      POSTGRES_HOST: ((POSTGRES_HOST))
+      POSTGRES_PASSWORD: ((POSTGRES_PASSWORD))
+      POSTGRES_USERNAME: ((POSTGRES_USERNAME))
```

Comment:
So this is where we must, must, must define env variables (even if we're going to fill in values in with the --var flag on the cf push command later).